src/Components/Counter.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import { db } from '../lib/firebase';
+
+const Counter = () => {
+  const [count, setCount] = useState();
+  const newCount = db.collection('count').doc('FQyeXUHAfge9AXp6qXrK');
+
+  useEffect(() => {
+    newCount
+      .get()
+      .then(function (doc) {
+        setCount(doc.data().number);
+      })
+      .catch((error) => {
+        console.log('Error getting document:', error);
+      });
+  }, [newCount]);
+
+  const increaseCount = (newCount) => {
+    newCount
+      .update({ number: count + 1 })
+      .then(() => {
+        setCount(count + 1);
```

Comment:
Just to capture what we discussed in our workshop: there's a way to reliably track your data without reusing `+ 1` as we do here. Don't let this be a blocker right now – you know you can update and read the state, and that's the main goal! We can always work on elegant solutions later.